### ISSUE TITLE
XGD_Runtime Error and Loading Video problem

### ISSUE THREAD

**Issue by memina** [neuralet/smart-social-distancing#25]
Hello, thank you to everyone who contributed to this project.
When I try it on Jetson nano, the video feels like loading continuously. It plays after 5 seconds, loads again, plays again for 5 seconds, then pauses. I refresh the page but the same problem continue. It also gives errors related to XGD_Runtime on console.

I created the docker image with the command below.
`docker build -f jetson-nano.Dockerfile -t "neuralet/smart-social-distancing:latest-jetson-nano" .`

Also I tried it on my PC, the result is same. What am i missing ? 

<img width="1323" alt="Screen Shot 2020-06-26 at 20 45 45" src="https://user-images.githubusercontent.com/12199743/86169699-b1883f80-bb22-11ea-88eb-b429edd4b5b1.png">

**Comment by JsonSadler** [comment]
Hi,
Thanks for your feedback :)

It's weird. Can you try running the docker without sudo?

**Comment by memina** [comment]
yes, but i can't run docker without sudo command.

**Comment by JsonSadler** [comment]
> yes, but i can't run docker without sudo command.
Try:
`sudo usermod -aG docker $USER`
Then logout/login or reboot.

**Comment by JsonSadler** [comment]
@memina Is this solved?

**Comment by nabil-bukhari** [comment]
I am having the same issue without any display 

![Screenshot from 2020-07-30 18-48-52](https://user-images.githubusercontent.com/68958543/88944393-59656a00-d295-11ea-9487-d7f9a85012cf.png)

**Comment by renzodgc** [comment]
Hey, we just got around checking these old issues, sorry about the delay.

The issue could be related to processing speed or the old frontend. Considering a few improvements have been done on the former and that we deprecated the old frontend for a new one on react, I presume this has been fixed. (Although the video streaming is still a bit rough around the edges)

Feel free to try to reproduce it if you are still interested and report back If performance is still an issue. Elsewise I'll close this issue in a week :)
